Gemfile

```diff
@@ -14,4 +14,8 @@ end
 # your gem to rubygems.org.
 
 # Load Gemfile with dependencies from manageiq
-eval_gemfile(File.expand_path("spec/manageiq/Gemfile", __dir__))
+MANAGEIQ_GEMFILE = File.expand_path("spec/manageiq/Gemfile", __dir__)
+unless File.exist?(MANAGEIQ_GEMFILE)
+  raise 'You must run `bin/setup` to install manageiq into spec/manageiq as a rails dummy app for this provider engine gem'
```

Comment:
@durandom Hmm... well, you might run into a "chicken or the egg" problem since this error would be raised when `bundler` is parsing the Gemfile, not after it has installed the gems prior to this line, meaning Rails or any other gems required by `bin/setup` might not be available to someone who is using a clean gemset or doing this on a fresh install of Ruby (FYI, as of writing this, I haven't looked at `bin/setup` for this repo... so I might be full of it).

Would this make more sense at the top of the `Rakefile` instead?  Then when someone tries running the tests, they will get the error there instead.